src/main/kotlin/com/developerphil/adbidea/action/EnableMobileAction.kt

```diff
@@ -0,0 +1,9 @@
+package com.developerphil.adbidea.action
+
+import com.developerphil.adbidea.adb.AdbFacade
+import com.intellij.openapi.actionSystem.AnActionEvent
+import com.intellij.openapi.project.Project
+
+class EnableMobileAction : AdbAction() {
```

Comment:
Couldn't get this command to work but it does show a notification that the command succeeded.

looks like the adb command is not properly formed and we just get a help command returned to us.

![image](https://user-images.githubusercontent.com/279299/78245973-5d83a580-74b6-11ea-84d5-2f8cabdef0d4.png)

Tested on an Android 9 emulator.

Reply:
I'll fix it in a few minutes, I might have just misspelled something :)

Reply:
Awesome, works perfectly, thanks !